src/index.css

```diff
@@ -0,0 +1,27 @@
+/* stylelint-disable */
+ body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+} 
```

Comment:
- [ ] Kindly make sure you follow the original template design in the requirements of this project by applying the same style(color, layout, font family, font size, margin, padding...

![image](https://user-images.githubusercontent.com/65068771/182011064-e74ae7a9-4184-4514-8723-016fdd7d9ce2.png)

- [ ] Kindly you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

- a description of the project.
- a demo of the project features.
- you should also highlight some interesting piece of code or something you built that you are very proud of.

- [ ] Kindly make sure also A link to the deployed project is present in the README.

Reply:
- [ ] Links to the live demo [deployed version], is still missing from the readme file
- [ ] As mentioned with the orevious reviewer, you should also record a video presenting your project, and add the link to the video in your reame file
- [ ] [optional] Last but not least I would advise you to work on your readme file so it looks professional, like make sure that all of the links there works, and also make sure to add these credits `The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.` to your readme file

Reply:
- [ ] Kindly make sure that you deploy this project and provide a live demo link in your readme file,
![Screenshot from 2022-07-31 08-31-08](https://user-images.githubusercontent.com/56886064/182013351-2dcf2b0d-8c87-4c9f-bd38-b63eaf26fd34.png)

Note: I WILL EXPLAIN MORE ABOUT THE LIVE DEMO LINK
=> It's a link which with which we click and we get to the live or deployed version of your project, you can deploy it on heroku or netlify or any other server, and then after deploying it, you will get a URL which will redirect to that page. This is a hard requirement to get this project complete.

- [ ] [optional] _I am marking this requirement as optional because it was not mentioned in the `first-review` but it should be a hard requirement as it is mentioned in the project description_ =>  =>  => I would advise you to work on your readme file so it looks professional, like make sure that all of the links there works, and also make sure to add these credits `The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.` to your readme file